src/middlewared/middlewared/plugins/chart_releases_linux/resources.py

```diff
@@ -303,3 +312,28 @@ async def get_consumed_host_paths(self):
             apps[app_name] = await self.middleware.call('chart.release.host_path_volumes', app_resources[app_name])
 
         return apps
+
+    @private
+    async def stats(self, release_name):
+        chart_release = await self.middleware.call('chart.release.get_instance', release_name, {
+            'extra': {'retrieve_resources': True}
+        })
+        return await self.stats_internal(chart_release['resources']['pods'])
+
+    @private
+    async def stats_internal(self, pods):
+        stats = {'memory': 0, 'cpu': 0, 'network': {'incoming': 0, 'outgoing': 0}}
+        for pod_stats in await self.middleware.call(
+            'k8s.stats.summary', [
+                ['podRef.name', 'in', [p['metadata']['name'] for p in pods]]
+            ]
+        ):
+            stats['cpu'] += int(pod_stats['cpu']['usageNanoCores'])
+            stats['memory'] += int(pod_stats['memory']['rssBytes'])
+            for interface in pod_stats['network']['interfaces']:
+                stats['network']['incoming'] += int(interface['rxBytes'])
+                stats['network']['outgoing'] += int(interface['txBytes'])
+
+        stats['cpu'] = ((stats['cpu'] / 1000000000) / await self.get_system_cores()) * 100
```

Comment:
I still don't understand why you're changing all of this and adding another duplicate method `get_system_cores`. Why not do:
```suggestion
        stats['cpu'] = ((stats['cpu'] / 1000000000) / (await self.middleware.call('system.cpu_info'))['core_count']) * 100
```

Reply:
@yocalebo that is to avoid doing `self.middleware.call` calls..I can change it to use this but this would be calling middleware quite a number of times and we can avoid it easily..?